Core/include/Acts/Material/interface/ISurfaceMaterialAccumulater.hpp

```diff
@@ -0,0 +1,64 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/Geometry/GeometryIdentifier.hpp"
+#include "Acts/Material/ISurfaceMaterial.hpp"
+#include "Acts/Material/MaterialInteraction.hpp"
+
+#include <map>
+#include <memory>
+#include <vector>
+
+namespace Acts {
+
+class Surface;
+
+/// @brief Interface for the material mapping, this is the accumulation step
+class ISurfaceMaterialAccumulater {
+ public:
+  /// @brief SurfaceAssignment is a surface, a position and a direction
+  using SurfaceAssignment = std::tuple<const Surface*, Vector3, Vector3>;
```

Comment:
Since this was converted to a struct in the previous one it should also be converted here.

Reply:
Weird, something with the rebase must have gone wrong ...

Reply:
Ah no, was not ... it's just necessary updates.

Reply:
Updated.